lib/bespoke.js

```diff
@@ -52,6 +52,11 @@ var from = function(selectorOrElement, plugins) {
       return eventData;
     },
 
+    destroy = function(customData) {
+      fire('destroy', createEventData(activeSlide, customData));
+      parent.parentNode.removeChild(parent);
```

Comment:
This should be a conditional in the case that the parent isn't attached to a DOM node (such as in a test).

```js
if (parent.parentNode) {
    parent.parentNode.removeChild(parent);
}
```